### PR TITLE
Makes revolution objectives softer

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

- Revs no longer block the shuttle
- Revs no longer have a deconversion message
- Lawyers and HOPs can now deconvert people by using a space law book on them. If the lawyer is a revolutionary, then this will act as a method of conversion instead.
- Revs no longer have objectives to kill all the heads of staff, instead its a soft objective to establish a new command structure which works the same way but doesn't really matter if someone fucks off to space since it won't affect if you win or lose
- Revs no longer ends when all the heads of either side are killed
- Completely removes the midround rev ruleset cause its just stupid

## Why It's Good For The Game

Why do we need to delay rounds and do all this crap just because a head went into space. Theres no climactic ending when the CE is in space and stalling the round. Just end the round, the events of the round were enough. The antagonist exists to invoke chaos, I don't really actually care if they win or not. Players can decide if they feel like they won or lost themselves which I am sure they will do.

## Testing Photographs and Procedure

image

image

image

## Changelog
:cl:
balance: Revs no longer block the shuttle
balance: Revs no longer have a deconversion message
balance: Lawyers and HOPs can now deconvert people by using a space law book on them. If the lawyer is a revolutionary, then they will use this to convert people instead.
balance: Revs no longer have objectives to kill all the heads of staff, instead its a soft objective to establish a new command structure which works the same way but doesn't really matter if someone fucks off to space since it won't affect if you win or lose
balance: Revs no longer ends when all the heads of either side are killed
balance: Completely removes the midround rev ruleset cause its just stupid
/:cl:

### CODE REVIEW

**Comment by Fronsis** [comment]
My questions:
- What about mindshields? Since it's not commented are they still the go-to to protect people against the rev conversion?

- Why remove the deconversion message? It's standard for revs and both cultists gamemode.

- How is Security expected to deal with the Head Revs which i assume are going to be inmune to deconversion.

- How does the Revolution ''win'' what is a ''new command structure'', looking at the code the End of Round  win condition messages are still equally vague as old revs and the proposed rework, seems like they'll win if they just.. stay alive? or a minor victory if the shuttle gets called (which is silly because it can get called by multiple things, they could've evacuated due to a space dragon but the revs ''got a minor victory'' out of that.

- What kind of conversion method these revs use, the old flash?
> Revs no longer block the shuttle
Perfect.

> Lawyers and HOPs can now deconvert people by using a space law book on them. If the lawyer is a revolutionary, then this will act as a method of conversion instead.
I mean.. sure i guess, they're the Chaplain of the rev gamemode.

> Revs no longer have objectives to kill all the heads of staff, instead its a soft objective to establish a new command structure which works the same way but doesn't really matter if someone fucks off to space since it won't affect if you win or lose

Well, it certainly implies that at least the heads of staff needs to die but if they escape it's whatever so i guess that's nice, i've posted my question about this above.

> Revs no longer ends when all the heads of either side are killed

Seems relevant to my fourth question, when is the gamemode exactly ''Over''? Like.. if they manage to kill all three head revs it makes sense to get the Centcom message of ''the mutiny has been quelled!'' The players need IC information to know when  the threats pass or is over so they can return to their work, lower alert level, etcetera otherwise it's just paranoia and mass-mindshield everyone, i highly suggest that if all three Head Revs die Centcom sends the message that the mutiny is over so the Crew, Command and Sec can move on, i'm obviously not expecting the Golden Era revs that literally ended the round of course.

> Completely removes the midround rev ruleset cause its just stupid

Please **no one** wants to suffer more midround revs, i beg you to implement it editing the file outside of this PR like we spoke on Discord instead of waiting for this to be reviewed, approved and merged/rejected.

**Comment by PowerfulBacon** [comment]
There is no end condition, there is no win/loss and the objective has no greentext/redtext. Revs don't need it since its more like wizard in that you just need to set people off.
I already did disable midround revs in the config.

**Comment by Fronsis** [comment]
> There is no end condition, there is no win/loss and the objective has no greentext/redtext. Revs don't need it since its more like wizard in that you just need to set people off. I already did disable midround revs in the config.

This reminds me of the ''''Optional objectives'''' that you were against back when you removed them, if the players don't have a simple ''Hey we won/lost'' then that's a flaw IMO, there needs to be clarity otherwise... What's the point? If there's no clear end goal achieved by the end other than a flimsy roundstart text of ''Make the station great again!'' it's just gonna be aimless, relying on mass-flash spam to tide and fight, you're letting a headless chicken loose on the station through the entire round, specially if you removed the notification that claims that the ''mutiny has been quelled'', could you elaborate on why that was removed and if possible adress my other questions?

File: code/game/objects/items/manuals.dm
Comment (Liver-1) [inline code/game/objects/items/manuals.dm:1]: "you are contractually obliged not commit them." Seems like a grammatical error.

**Comment by ColonelOrion** [comment]
So just to be clear; can I still throw improvised explosives at security in this game mode?

I like the riot vibe.

File: code/game/objects/items/manuals.dm [hunk 1]
```diff
@@ -299,6 +299,53 @@
...
+		return
+	if (user.mind?.has_antag_datum(/datum/antagonist/rev))
+		var/datum/antagonist/rev/rev = user.mind.has_antag_datum(/datum/antagonist/rev)
+		user.say("Isn't it obvious, Nanotrasen, the governments; everyone around us has been tricking us, playing us like we are pawns...", forced = "space law")
+		if (rev.add_revolutionary(target.mind, FALSE))
+			target.visible_message("<span class='notice'>[target] nods in approval, taking in the information!</span>", "<span class='notice'>That all makes perfect sense, the truth washes over you!</span>")
+		else
+			target.visible_message("<span class='userdanger'>[target] spits on the floor, disrespecting [user]'s authority!</span>", "<span class='notice'>You finish listening to [user]'s waffling. What a knobhead, you think to yourself...</span>")
+	else
+		user.say("These shall all be considered acts which are in violation of your contract of employment, and you are contractually obliged not commit them.", forced = "space law")
```
Comment (PowerfulBacon) [inline code/game/objects/items/manuals.dm]:
```suggestion
		user.say("These shall all be considered acts which are in violation of your contract of employment, and you are contractually obliged to not commit them.", forced = "space law")
```

**Comment by Rukofamicom** [comment]
Damn labels

**Review by Rukofamicom — APPROVED** [review]
This is still testmerged technically because the only thing stopping me from fully merging it is the fact it needs wiki changes and I am too lazy to make them myself.

**Comment by PowerfulBacon** [comment]
Wiki changes done, this is more a temporary measure but we'll see how it goes

**Pull request merged**